packages/gatsby-cli/README.md

```diff
@@ -10,7 +10,7 @@ package installed locally.
 
 ## Install
 
-`npm install --global gatsby-cli`
+`npm install --global gatsby-cli react react-dom`
```

Comment:
I don't think that's right. We don't want to install those globally.

Reply:
womp, thank you

Reply:
I'm not expert on dependencies but these propably should be added to starters as dependencies but there are so many of those starters and many of them are unofficial so making react and react-dom peer dependencies will break them I think

Reply:
I just arrived at the same conclusion 👍 

This is coming with a major version bump – I'd be surprised if this is the only thing that breaks compatibility with existing starters, but I don't know

Reply:
yeah — we'll upgrade the official starters and perhaps a few others. There'll be an upgrade guide for sites that starters will need to go through. It won't break anything though as a starter with a gatsby@1 dependency will keep working as normal.